.yarn/patches/@metamask-accounts-controller-npm-11.0.0-c67f9a2901.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/dist/utils.js b/dist/utils.js
+index 0a77dd709e786ee586b70677fa951f34d5097e34..24dbc12c31686a3ad1b78de702801e19a239f0fc 100644
+--- a/dist/utils.js
++++ b/dist/utils.js
+@@ -11,11 +11,11 @@ const uuid_1 = require("uuid");
+  * @returns The name of the keyring type.
+  */
+ function keyringTypeToName(keyringType) {
+     // Custody keyrings are a special case, as they are not a single type
+     // they just start with the prefix `Custody`
+-    if ((0, keyring_controller_1.isCustodyKeyring)(keyringType)) {
```

Comment:
Do we need this patch at all anymore? I believe that the `isCustodyKeyring` check from `keyring-controller` already handles this case (thanks to https://github.com/MetaMask/core/pull/3899, which became available in `@metamask/accounts-controller` 11.0.0).

Reply:
I removed everything else but kept this because I was seeing a related error at one point. Let me check again.

Reply:
Removing this doesn't seem to cause any issues on unit tests! I'll get rid of it if the CI run passes.

Reply:
Removed here! b825ed40a3af4d7b4f4f2cd4a462b1bb37b78477